README.md

```diff
@@ -69,7 +71,7 @@ Launch file `natnet_ros.launch` contains the several configurable arguments. The
 
 #### Publishing the single marker 
 It is possible to track the single marker as a rigid body with constant orientation. Go to the `config/initiate.yaml` It is suggested to make a copy of the file and rename the new file.
-The file contains the details on what to modify. 
+The file contains the details on what to modify. Basically, the approximate position of an individual marker has to be noted in this file and a basic filtering algorithm will track it over time. The filtering is not that robust and you can see it disappear often
```

Comment:
Improved algorithm will be available soon. I am working on major changes and some additional features.